source/Calamari/Program.cs

```diff
@@ -67,7 +68,7 @@ internal int Run(CommonOptions options)
             {
```

Comment:
Would it be hard to make this inherit from `CalamariFlavourProgram` so that we don't duplicate all the start up code?